app/src/main/java/net/squanchy/navigation/Navigator.java

```diff
@@ -26,51 +27,48 @@
 
 public class Navigator {
 
-    private final Context context;
+    private static final int NO_FLAGS = 0;
```

Comment:
...yes there should. Will fix in next PR, takes too much to do a CI build for this XD